Adding CI feature with .travis.yml file

carmat88 opened the pull request:
File .travis.yml is being created in order to add the CI feature to the docs repository and testing correctness of documentation

Commit:
Adding CI feature with .travis.yml file

Commit:
.travis.yml corrected with a missing space at the end

mcapuccini commented on .travis.yml:
```diff
@@ -0,0 +1,15 @@
+language: python # (Ansible)
+
+sudo: required
```
Try to see if you can avoid sudo, so it will run faster

carmat88 replied:
Indeed, it seems that by removing sudo the test is faster

mcapuccini commented on .travis.yml:
```diff
@@ -0,0 +1,15 @@
+  # Install Sphinx
+  - sudo pip install --upgrade pip
+  - sudo pip install sphinx==1.5.3
+  - sudo pip install sphinx-autobuild==0.6.0
```
plase try to remove this

carmat88 replied:
same here, sudo removed from the pip commands

mcapuccini commented on .travis.yml:
```diff
@@ -0,0 +1,15 @@
+  - sudo pip install --upgrade pip
+  - sudo pip install sphinx==1.5.3
+  - sudo pip install sphinx-autobuild==0.6.0
+  - sudo pip install sphinx_rtd_theme==0.2.4
```
please try to review this

carmat88 replied:
same here, sudo removed

mcapuccini reviewed (changes requested):
Some minor things need to be fixed/investigated

carmat88 committed:
Update bootstrap.rst

Updating line 36 with an extra ~ (due to Sphinx processing error via Travis CI)

carmat88 committed:
Update bootstrap.rst

Updating line 110 with an extra ~

Commit:
Correction of various warnings being thrown by Sphinx via Travis CI

Commit:
removed requirement of sudo from travis.yml file

carmat88 commented:
CI feature is correctly working
